packages/puppeteer/tsconfig.json

```diff
@@ -1,24 +1,4 @@
 {
-  "compilerOptions": {
-    /* Basic Options */
-    "target": "esnext",
-    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "lib": [
-      "dom",
-      "esnext"
-    ] /* Specify library files to be included in the compilation. */,
-    "declaration": true /* Generates corresponding '.d.ts' file. */,
-    "sourceMap": true /* Generates corresponding '.map' file. */,
-    "outDir": "dist" /* Redirect output structure to the directory. */,
-    "rootDir": "src" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
-    "removeComments": true /* Do not emit comments to output. */,
-    "pretty": true,
-    "strict": true,
-    "noImplicitAny": true,
-    "noImplicitThis": true,
-    "noImplicitReturns": true,
-    "moduleResolution": "node",
-    "esModuleInterop": true
-  },
+  "extends": "../../tsconfig.json",
```

Comment:
This is hiding a change from `module: commonjs` to `module: esnext` that probably shouldn't be part of this PR